src/main/java/org/cbio/gdcpipeline/util/CommonDataUtil.java

```diff
@@ -110,40 +110,6 @@ public String toString() {
         }
     }
 
-    public static List<File> getFileList(List<Hits> gdcFileMetadatas, CommonDataUtil.GDC_TYPE type, String sourceDir) {
-        List<File> fileList = new ArrayList<>();
-        List<File> compressedFiles = new ArrayList<>();
-        if (!gdcFileMetadatas.isEmpty()) {
-            for (Hits data : gdcFileMetadatas) {
-                if (data.getType().equals(type.toString())) {
-                    File file = new File(sourceDir, data.getFile_name());
-                    if (file.exists()) {
-                        if (isCompressedFile(file)) {
-                            compressedFiles.add(file);
-                        } else {
-                            fileList.add(file);
-                        }
-                    } else {
-                        if (LOG.isInfoEnabled()) {
-                            LOG.info(type.toString() + " file : " + file.getAbsolutePath() + " not found.\nSkipping File");
-                        }
-                    }
-                }
-            }
-        }
-        if (!compressedFiles.isEmpty()) {
-            try {
-                fileList.addAll(extractCompressedFiles(compressedFiles));
-            } catch (Exception e) {
-                e.printStackTrace();
-                if (LOG.isErrorEnabled()) {
-                    LOG.error("Skipping files to extract.");
-                }
-            }
-        }
-        return fileList;
-    }
-
     public static List<File> extractCompressedFiles(List<File> fileList) throws Exception {
         temp_dir = createTempDirectory();
```

Comment:
is this done with the java file utils? the tmp directory path / creation? I didn't dig into where `createTempDirectory` is coming from but it doesn't look like the user can specify where temp dir path.  When we run the cbio importer we set the temp dir path as a command line arg `-Djava.io.tmpdir=some/tmp/dir`

Reply:
It looks like it is. The dir name is set though.
`private static String SYSTEM_TMP_DIR_PROPERTY = "java.io.tmpdir";`
`private static String TMP_DIR_NAME = "gdcpipeline";`
`File tmp_dir = new File(System.getProperty(SYSTEM_TMP_DIR_PROPERTY), TMP_DIR_NAME);`